### ISSUE TITLE
JScrollPane and Ajax

### ISSUE THREAD

**Issue by joelmatisa** [vitch/jScrollPane#379]
Good morning all,
I don't know if it's me but I noticed some pretty weird behavior, when I fetch the data via ajax and I made a plugin call: $ ('. Scroll-user-table'). JScrollPane ( {
       showArrows: true,
       hijackInternalLinks: true,
       autoReset: true,
       horizontalArrowPositions: 'after',
     });
After there is a jump when I try to click on an element of the content taken via ajax
It works when I click for the second time

**Comment by illuusio** [comment]
Can you make some example site for this I can see the 'jump' as I can't reproduce.

**Comment by joelmatisa** [comment]
Just for precision this jump, I do when I scroll, as if I lose my hand on my dom

**Comment by illuusio** [comment]
> Just for precision this jump, I do when I scroll, as if I lose my hand on my dom

Sorry as non-native speaker I didn't quiet get what you mean? Is problem that calculation of line is or place is not correct..

**Comment by illuusio** [comment]
Ok looks like the problem that I tried to solve with last RC. Do you have small HTML for debug?

**Comment by joelmatisa** [comment]
This is the code I am loading via ajax, so the link and select (tag) at the bottom is not working well (as described above).

<div class="scroll-user-table overflow-auto">
  <table class="table-sortable relative w-full table-auto text-left">
      <thead class="border-b border-black">
        <tr class="text-xs font-bold">
            <th nowrap class="text-center px-4 py-2">Code commande</th>
            <th nowrap class="text-center px-4 py-2">Date commande</th>
            <th nowrap class="text-center px-4 py-2">Heure commande</th>
            <th nowrap class="text-center px-4 py-2">Nombre d'articles</th>
            <th nowrap class="text-center px-4 py-2">Montant panier</th>
            <th nowrap class="text-center px-4 py-2">Montant livraison</th>
            <th nowrap class="text-center px-4 py-2">Paiement total commande</th>
            <th nowrap class="text-center px-4 py-2">Moyen de paiement</th>
            <th nowrap class="text-center px-4 py-2">Statut paiement</th>
            <th nowrap class="text-center px-4 py-2">Livreur</th>
            <th nowrap class="text-center px-4 py-2">Type de livraison</th>
            <th nowrap class="text-center px-4 py-2">Statut livaison</th>
            <th nowrap class="text-center px-4 py-2">Date Récupéré</th>
            <th nowrap class="text-center px-4 py-2">Heure Récupéré</th>
            <th nowrap class="text-center px-4 py-2">Date Livré</th>
            <th nowrap class="text-center px-4 py-2">Heure Livré</th>
            <th nowrap class="text-center px-4 py-2">Client </th>
            <th nowrap class="text-center px-4 py-2">Marchand</th>
            <th nowrap class="text-center px-4 py-2">Options</th>
            <th nowrap class="text-center px-4 py-2">Code commande</th>
            <th nowrap class="text-center px-4 py-2">Date commande</th>
            <th nowrap class="text-center px-4 py-2">Heure commande</th>
            <th nowrap class="text-center px-4 py-2">Nombre d'articles</th>
            <th nowrap class="text-center px-4 py-2">Montant panier</th>
            <th nowrap class="text-center px-4 py-2">Montant livraison</th>
            <th nowrap class="text-center px-4 py-2">Paiement total commande</th>
            <th nowrap class="text-center px-4 py-2">Moyen de paiement</th>
            <th nowrap class="text-center px-4 py-2">Statut paiement</th>
            <th nowrap class="text-center px-4 py-2">Livreur</th>
            <th nowrap class="text-center px-4 py-2">Type de livraison</th>
            <th nowrap class="text-center px-4 py-2">Statut livaison</th>
            <th nowrap class="text-center px-4 py-2">Date Récupéré</th>
            <th nowrap class="text-center px-4 py-2">Heure Récupéré</th>
            <th nowrap class="text-center px-4 py-2">Date Livré</th>
            <th nowrap class="text-center px-4 py-2">Heure Livré</th>
            <th nowrap class="text-center px-4 py-2">Client </th>
            <th nowrap class="text-center px-4 py-2">Marchand</th>
            <th nowrap class="text-center px-4 py-2">Options</th>
            <th nowrap class="text-center px-4 py-2">Code commande</th>
            <th nowrap class="text-center px-4 py-2">Date commande</th>
            <th nowrap class="text-center px-4 py-2">Heure commande</th>
            <th nowrap class="text-center px-4 py-2">Nombre d'articles</th>
            <th nowrap class="text-center px-4 py-2">Montant panier</th>
            <th nowrap class="text-center px-4 py-2">Montant livraison</th>
            <th nowrap class="text-center px-4 py-2">Paiement total commande</th>
            <th nowrap class="text-center px-4 py-2">Moyen de paiement</th>
        </tr>
      </thead>

      <tbody class="text-sm tbody-category" id="tbody-category">
        <td nowrap class="px-4 py-2">110-00-4</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">110-00-4</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">110-00-4</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">110-00-4</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">110-00-4</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">110-00-4</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">110-00-4</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">110-00-4</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">110-00-4</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">110-00-4</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">110-00-4</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">110-00-4</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">110-00-4</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">110-00-4</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">110-00-4</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">110-00-4</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">110-00-4</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">110-00-4</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">110-00-4</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2">110-00-4</td>
        <td nowrap class="px-4 py-2">Lorem ipsum dolor sit</td>
        <td nowrap class="px-4 py-2"><a href="https://www.google.com" class="">google</a></td>  
        <td nowrap class="px-4 py-2">
          <select>
            <option>1</option>
            <option>2</option>
          </select>
        </td> 
      </tbody>
  </table>
</div>

**Comment by illuusio** [comment]
Thanks I try to get time to take a look this next week or after that.

**Comment by joelmatisa** [comment]
Thank you come back to me soon

**Comment by joelmatisa** [comment]
Hello @illuusio, I still haven't had a follow-up on our last exchange

**Comment by illuusio** [comment]
Yes you are correct. I was more optimistic about this week and everything didn't go as I planned so this will be dealed in couple weeks. This is kind of hobby to maintain this so time what I can spend with this is very limited.

**Comment by Gopu17** [comment]
Hi @illuusio @joelmatisa , I got the same issue after I upgrade JQuery from v3.2.1 to v3.7.1, could you please provide any solution for this?
